drivers/clock_control/clock_control_r8a779f_cpg_mssr.c

```diff
@@ -68,12 +68,12 @@ static int r8a779f_cpg_enable_disable_core(const struct r8a779f_cpg_mssr_cfg *cf
 	case R8A779F_CLK_SD0CKCR1:
 		return 0;
 	case R8A779F_CLK_SD:
-		reg = sys_read32(cfg->base_address + clk_info->offset);
+		reg = sys_read32(DEVICE_MMIO_GET(dev) + clk_info->offset);
```

Comment:
nit: value of `DEVICE_MMIO_GET` probably can be assigned to base_address on init to reduce number of changes in this commit

Reply:
I prefer to leave only one base address inside structure